tests/rbac_test.go

```diff
@@ -147,6 +149,34 @@ var _ = Describe("Aggregated role in-action tests", Serial, func() {
 		Entry("[test_id:3949]can do everything with edit", "edit"),
 	)
 
+	DescribeTable("check all user facing resources can be manipulated by non-cluster-admin", func(user string) {
+		var namespacedCDIAPIResourcesPlural []string
+		// Also retrieves API resources that are not served via CRDs
+		outputAPIResources, err := f.RunKubectlCommand("api-resources", "--namespaced", "-o", "name")
+		Expect(err).ToNot(HaveOccurred(), "ERR: %s, OUT: %s", err, outputAPIResources)
+		for _, apiResource := range strings.Split(strings.TrimSpace(outputAPIResources), "\n") {
+			if strings.Contains(apiResource, "cdi.kubevirt.io") {
```

Comment:
maybe use `HasSuffix` instead of `Contains`. limits the matches a bit.

Reply:
Makes sense, updated for now